lms/static/js/groups/views/cohorts.js

```diff
@@ -304,7 +304,6 @@
                                 });
                             }
                         }).render();
-                        this.$('#file-upload-form-file').focus();
```

Comment:
I don't think it's necessary here. The next tab stop after activating the "tab" would be this field anyway, so this could be confusing to some.